lib/matplotlib/tests/test_axes.py

```diff
@@ -1439,9 +1439,7 @@ def bump(a):
             x = 1 / (.1 + np.random.random())
             y = 2 * np.random.random() - .5
             z = 10 / (.1 + np.random.random())
-            for i in range(m):
-                w = (i / float(m) - y) * z
-                a[i] += x * np.exp(-w * w)
+            a += x * np.exp(-((np.arange(m) / m - y) * z) ** 2)
         a = np.zeros((m, n))
         for i in range(n):
             for j in range(5):
```

Comment:
`stackplot_demo2` (from which this function comes) has the slightly more helpful docstring "Return *n* random Gaussian mixtures, each of length *m*."  I can copy it there, or just inline the function (also in the demo), let me know if you have a preference.